randomizer/Logic.py

```diff
@@ -647,7 +647,7 @@ def HintAccess(self, location, region_id):
             return False
         # The only weird exception: vanilla Fungi Lobby hint doors only check for Chunky, not the current Kong, and all besides Chunky's needs grab
         if not self.settings.wrinkly_location_rando and not self.settings.remove_wrinkly_puzzles and region_id == RegionEnum.FungiForestLobby:
-            return self.chunky and (location.kong == Kongs.chunky or (self.donkey and self.grab))
+            return (self.chunky or self.settings.wrinkly_available) and (location.kong == Kongs.chunky or (self.donkey and self.grab))
```

Comment:
The original is very weird but correct. All you need is Chunky (plus DK and grab) to be able to read all the hint doors in vanilla Fungi hint doors.

Reply:
Yes, but if you have kongless hint doors enabled you don't need Chunky to be unlocked in order to read the hints. Any kong (and if not the chunky door, DK and grab) will do with kongless hint doors. That's what I'm getting at in this change.

Reply:
Appearantly kongless hint doors doesn't affect Chunky's door on the wheel. Going to err on the side of caution and revert this change.